duckduckgo/Android
# Improve tab switcher ui

Reviewer: @subsymbolic 
Asana: https://app.asana.com/0/0/361925093999567

**Description**:
 Improved tab switcher UI following latest iOS design on Zeplin


**Steps to test this PR**:
1. Create a tab and open the Tab Switcher
2. Confirm that the tab switcher UI has changed


###### Reviewer Checklist:


###### PR DRI Checklist:
